### ISSUE TITLE
1.16.3 Newcomer multiplier does not take handler.yml into account

### ISSUE THREAD

**Issue by x1p** [kangarko/ChatControl-Pro#2092]
Settings.yml:
```
Newcomer:
  Threshold: 120 minutes
  Warn_Points_Multiplier: 2.0
```
Situation:
newcomer comes online and swears.
Swear word is matched, `handle as vulgar`
Handlers.yml:
```
vulgar:
  Warn_Points: swear 2
```
So in theory. this player should receive `2 swear` points, multiplied by `2.0` makes `4 swear` points.
However he only scores 2 swear points.

Apparently the newcomer function does not multiply the final points given by the handlers file but instead is limited to the base category.

**Comment by kangarko** [comment]
Thanks, this will be solved for CCP 9.

**Comment by x1p** [comment]
This issue is closed however it is not noted as "fixed" in 8.9.0 beta. "Before CHC 9" is very flexible ;-) I'm confused.

**Comment by kangarko** [comment]
I said we'll aim to fix that for the next generation ChatControl. Our old system was unfortunately poorly designed so I apology for that.
